build.sc

```diff
@@ -431,6 +431,8 @@ trait Core extends ScalaCliSbtModule with ScalaCliPublishModule with HasTests
          |  def workspaceDirName = "$workspaceDirName"
          |  def projectFileName = "$projectFileName"
          |  def jvmPropertiesFileName = "$jvmPropertiesFileName"
+         |  def scalacArgumentsFileName = "scalac.args.txt"
+         |  def maxScalacArgumentsCount = 5000
```

Comment:
unfortunately I'm  affraid this will be too low for our use case, sice we codegen tens of thousands of files in some modules

Reply:
This is just a limit above which arguments are passed as a `@<file>`

Reply:
Sorry, I've misread the implementation. Thank you for the fix! :)